meta-genivi-dev/recipes-sota/aktualizr/aktualizr_git.bb

```diff
@@ -12,19 +12,27 @@ S = "${WORKDIR}/git"
 
 SRCREV = "a6136f091d58b1bb97ec0dd1215eb570fa853f30"
 
-FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
-
 SRC_URI = " \
           git://github.com/advancedtelematic/aktualizr;branch=fix/boost-yocto \
+          file://aktualizr.service \
           "
 
+SYSTEMD_PACKAGES = "${PN}"
+SYSTEMD_SERVICE_${PN} = "aktualizr.service"
+
 DEPENDS = "boost curl openssl jansson dbus"
 RDEPENDS_aktualizr = "dbus-lib"
```

Comment:
Yes, according to the [mega manual ](http://www.yoctoproject.org/docs/latest/ref-manual/ref-manual.html#ref-classes-systemd) there is no need of SYSTEMD_PACKAGES if the service is in the main package. The name of the service file should be in SYSTEMD_SERVICE.